src/codegen/doc/format.rs

```diff
@@ -71,7 +71,10 @@ fn code_blocks_transformation(
 fn get_language<'a>(entry: &'a str, out: &mut String) -> &'a str {
     if let (_, Some(after)) = try_split(entry, LANGUAGE_SEP_BEGIN) {
         if let (before, Some(after)) = try_split(after, LANGUAGE_SEP_END) {
-            out.push_str(&format!("\n```{}", before));
+            out.push_str(&format!(
+                "\n\n**⚠️ The following code is in {0} ⚠️**\n\n```{0}",
```

Comment:
I don't think it's needed here. Because often those might be just plain types or functions. Worth trying it locally

Reply:
I'll add a check for the language itself, but even so, still needs to be marked as such. ;)

Reply:
But those can be sometimes replaced by the docs links no?

Reply:
Wait I'm lost now. In which case a \`\`\` code block can be replaced by a link?

Reply:
I guess when it's a one-line example?

Can you create a gtk-rs PR with this? Then we can check the generated docs :)

Reply:
We don't have the docs in the different repositories, so it wouldn't change anything.

Reply:
It's built as part of the github pages

Reply:
It would require to merge the PR for that. :)

Reply:
Are you sure? It should be available under https://guillaumegomez.github.io/gtk-rs-core/ and similar.

Reply:
Not sure about that. :-/